resources/lib/espnplayer.py

```diff
@@ -124,11 +124,26 @@ def get_channels(self):
         data = self.make_request(url, method='get', params=params)
         return data
 
-    def get_pkan(self, airing_id):
+    def get_espntoken(self, airing_id):
+        """Return a token needed to request a pkan/stream"""
+        url = 'https://www.espnplayer.com/secure/espntoken'
+        payload = {
+            'airingId': airing_id,
+            'format': 'json',
+        }
+        sc_data = self.make_request(url=url, method='post', payload=payload)
+        return json.loads(sc_data)['data']
```

Comment:
make_request() is already decoding the JSON now, so this should just return the sc_data directly.